tests/foreman/ui/test_host.py

```diff
@@ -259,6 +259,21 @@ def remove_vm_on_delete(target_sat, setting_update):
     yield
 
 
+@pytest.fixture
+def prepared_host(rex_contenthost, target_sat):
```

Comment:
Non-blocking, would suggest renaming this fixture as `prepared_host` seems something general.

Reply:
Yes.